From 7a35e760e83e11ebecb64fca8d5ca042c8adcac0 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 1 Jan 2021 00:31:37 +0100
Subject: [PATCH 1/1] doc: board: fix README.b4860qds

Two sibling headings can never be the same.

Fix the heading levels.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
Reviewed-by: Bin Meng <bmeng.cn@gmail.com>
---
 doc/board/freescale/b4860qds.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/board/freescale/b4860qds.rst b/doc/board/freescale/b4860qds.rst
index 37d7d08b09..de14d857b9 100644
--- a/doc/board/freescale/b4860qds.rst
+++ b/doc/board/freescale/b4860qds.rst
@@ -134,7 +134,7 @@ B4860QDS Default Settings
 -------------------------
 
 Switch Settings
----------------
+^^^^^^^^^^^^^^^
 
 .. code-block:: none
 
@@ -167,7 +167,7 @@ B4420QDS Default Settings
 -------------------------
 
 Switch Settings
----------------
+^^^^^^^^^^^^^^^
 
 .. code-block:: none
 
-- 
2.29.2

